### DISCUSSION TITLE
Please Approval of WalletConnect registry 🙏

### DISCUSSION THREAD

**Discussion by SlopeFinance** [WalletConnect/walletconnect-monorepo#1071] [Developer Support] [answered]
Hi there!
We have submitted an application to register our wallet "Slope Wallet"
Would you be so kind as to approve the application so that our users can access WallectConnect as soon as possible.
Thanks in advance.

**Comment by chadyj** [comment, marked as answer]
Approved :)

**Reply by SlopeFinance** [reply]
Thanks 🙏
